app/src/androidTest/java/com/android/sample/screen/SwipePageTest.kt

```diff
@@ -116,8 +132,12 @@ class SwipePageTest : TestCase() {
   @Test
   fun recipeAndDescriptionAreCorrectlyDisplayed() = runTest {
     composeTestRule.onNodeWithTag(RECIPE_NAME, useUnmergedTree = true).assertIsDisplayed()
-    composeTestRule.onNodeWithTag(RECIPE_STAR, useUnmergedTree = true).assertIsDisplayed()
-    composeTestRule.onNodeWithTag(RECIPE_RATE, useUnmergedTree = true).assertIsDisplayed()
```

Comment:
Nice catch, these values needed to be deleted before the M3, as we say in french *Chapeau* 👏